### PR TITLE
Refactoring: remove `ConsoleHelper`, set IO for services in controller action

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Fixed issues  | -

### CODE REVIEW

**Review by terabytesoftw — APPROVED** [review]

**Comment by terabytesoftw** [comment]
The idea of `ConsoleHelper::class` was to eliminate the duplicate code that is now everywhere.

**Comment by vjik** [comment]
> The idea of `ConsoleHelper::class` was to eliminate the duplicate code that is now everywhere.

`ConsoleHelper` create own input/ouput instead use input/output from console command. This PR fixed this.

File: src/Service/Database/ListTablesService.php [hunk 1]
```diff
@@ -4,65 +4,78 @@
...
         $dbname = $this->getDsnAttribute('dbname', $dsn);
 
-        $this->consoleHelper->io()->section("List of tables for database: {$dbname}");
+        $this->io->section("List of tables for database: {$dbname}");
 
         $count = 0;
 
-        $this->consoleHelper->table()->setHeaders(['Nº', 'Table']);
+        $table = new Table($this->io);
+        $table->setHeaders(['Nº', 'Table']);
```
Comment (samdark) [inline src/Service/Database/ListTablesService.php:69]:
```suggestion
        $table->setHeaders(['#', 'Table']);
```

File: src/Service/Migrate/DownService.php [hunk 2]
```diff
@@ -37,29 +42,37 @@ public function __construct(
      */
     public function run(string $class): bool
     {
-        $this->consoleHelper->io()->title("\nReverting $class");
+        if ($this->io) {
```
Comment (samdark) [inline src/Service/Migrate/DownService.php:45]: Is it possible to avoid this and alike `if`?

**Review by samdark — APPROVED** [review]

File: src/Service/Migrate/DownService.php (hunk 2 above)
Comment (vjik) [reply src/Service/Migrate/DownService.php:45]: Potentialy user of package can use `DownService` not in console. Output to console - optional.

**Comment by samdark** [comment]
👍

**Pull request merged**